src/codeu/chat/client/commandline/Chat.java

```diff
@@ -417,6 +418,8 @@ public void invoke(List<String> args) {
         System.out.println("    List all messages in the current conversation.");
         System.out.println("  m-add <message>");
         System.out.println("    Add a new message to the current conversation as the current user.");
+        System.out.println("  u-authorize <user> <authority>");
```

Comment:
Hey, I'm just gonna leave some suggestions here to help you, so once again, this is a premature code review that doesn't fully count. Whenever you're on one menu of commands in this app, they all start with the same letter: u-menu is when you boot up, c-menu once you login, m-menu for conversation commands, s-menu for status update. So this should be m-authorize just to keep everything organized. Also, please explain authority markers in this dialog.